src/index.js

```diff
@@ -88,11 +98,11 @@ async function serviceembed(params, url, log) {
           'Content-Type': 'text/html',
           'Cache-Control': `max-age=${json.cache_age ? json.cache_age : '3600'}`,
         },
-        body: `<div class="embed embed-oembed embed-advanced">${json.html}</div>`,
+        body: `<div class="embed embed-${kind} embed-oembed embed-advanced">${json.html}</div>`,
```

Comment:
As suggested in #211, I would remove `embed-advanced` since nobody knows what it's for.